### DISCUSSION TITLE
How to get source code

### DISCUSSION THREAD

**Discussion by hanyu1980** [HewlettPackard/swarm-learning#173] [General]
Hi everyone! 

I want to study swarm learning source code, how to get source code? or how to buy source code?

**Comment by iArpanPatel** [comment]
This has been answered on our [HPE Developer Slack Workspace](https://slack.hpedev.io/)  [#hpe-swarm-learning](https://hpedev.slack.com/archives/C04A5DK9TUK) channel.
